dissect/target/plugins/os/windows/datetime.py

```diff
@@ -76,6 +76,9 @@ def parse_systemtime_transition(systemtime: c_tz._SYSTEMTIME, year: int) -> date
     Reference:
         - https://docs.microsoft.com/en-us/windows/win32/api/timezoneapi/ns-timezoneapi-time_zone_information
     """
+    if systemtime.wDay > 5:
```

Comment:
It can't be lower than `1` either, why only specifically check for this one? Either check for everything or not at all I think?